internal/workflow_testsuite.go

```diff
@@ -262,6 +262,16 @@ func (t *TestActivityEnvironment) SetOnActivityHeartbeatListener(
 	return t
 }
 
+// SetActivityStartToCloseTimeout sets the start to close timeouts of activities in the test environment.
+func (e *TestActivityEnvironment) SetActivityStartToCloseTimeout(timeout time.Duration) {
+	e.impl.testWorkflowEnvironmentShared.activityStartToCloseTimeout = timeout
+}
+
+// SetActivityStartToCloseTimeout sets the schedule to close timeouts of activities in the test environment
+func (e *TestActivityEnvironment) SetActivityScheduleToCloseTimeout(timeout time.Duration) {
+	e.impl.testWorkflowEnvironmentShared.activityScheduleToCloseTimeout = timeout
+}
```

Comment:
Hrmm, longer than 10-minutes using this framework is a bit rough. You might consider a proper integration test with a real server instead.

Regardless, I wonder if we should document on these what the default is and document any relation to `SetTestTimeout` here and on that method.

Reply:
Yeah we should include a comment here on what the default values are